src/valora-dapp-list.json

```diff
@@ -29,6 +35,14 @@
       "description": "dapps.moola",
       "logoUrl": "https://raw.githubusercontent.com/valora-inc/app-list/main/assets/moola.png",
       "url": "https://app.moola.market/"
+    },
+    {
+      "name": "Bidali",
+      "id": "3",
+      "categoryId": "3",
+      "description": "dapps.bidali",
+      "logoUrl": "https://raw.githubusercontent.com/valora-inc/app-list/main/assets/bidali.png",
+      "url": "https://www.bidali.com/"
```

Comment:
I'm not sure about this though. Anyone can confirm?

Reply:
Hmm, I think this one should go to the in-app screen, not be open in an external browser 🤔